src/App.jsx

```diff
@@ -1,7 +1,101 @@
+import { useState } from "react";
```

Comment:
```suggestion
import { useState, useEffect } from "react";
```
8번 라인에 import `useEffect`가 너무 외로워 보이는데 여기에 같이 써주는건 어떨까요??🥹

Reply:
앗 ㅋㅋㅋㅋㅋㅋ useEffect랑 useState랑 싸워서 따로 놔달랬어요 ㅋ
장난이고 좋네요 ㅎㅎ 항상 임포트문 정리 잘 안해서 피드백 달리곤 했었는데 또 그랬따..

Reply:
import문 관리가 제일 귀찮은 것 같네요.. 막상 하면 깔끔하고 기분 좋은데 하기가 귀찮은 느낌..?

그래도 eslint 플러그인 중에 `eslint-plugin-import` 이거 사용하면 프레임워크, 라이브러리 등의 기준으로 나눠주고 이름 오름차, 내림차순 정렬 같은 것도 되서 나중에 한번 써보셔도 좋을 것 같습니다🙃

Reply:
띠용? 이런 꿀정보를;; 감사합니당 바로 써볼게요 ㅎㅎㅎㅎ